index.js

```diff
@@ -36,8 +38,8 @@ db.sequelize
   })
   .then(startServer)
   .catch((err) => {
-      if (err) debug('An error occured %j', err);
-      else debug('Database synchronized');
+      if (err) logger.debug('An error occured %j', err);
```

Comment:
NB: The string templating in `%j` won't happen because you're now using structured logs.

You fixed in https://github.com/amida-tech/amida-messaging-microservice/pull/63/files#diff-168726dbe96b3ce427e7fedce31bb0bcR32

One of the downsides of merging in multiple commits; small differences. :woman_shrugging: